### ISSUE TITLE
Client adds can't be clicked/closed on OSX Sierra 10.12.2 

### ISSUE THREAD

**Issue by ArtificialPB** [powerbot/powerbot#1572]
Switching input off and on does not fix the issue.

<img width="773" alt="screen shot 2017-01-09 at 10 54 28" src="https://cloud.githubusercontent.com/assets/4672155/21762678/586993dc-d65a-11e6-8246-ca443d727593.png">

**Comment by ArtificialPB** [comment]
It seems that the RSBot frame is blocking the ad. When you move the frame, the ad stays fixed on the screen and can be clicked when the frame is out of the way.

**Comment by Paris** [comment]
Have you tried updating your OS X? I'm on 10.12.4 and it works fine.

**Comment by ArtificialPB** [comment]
Yes, I'm running the latest version (10.12.4). It doesn't happen every time though.

**Comment by ArtificialPB** [comment]
Here's a pic of when it happens:
<img width="975" alt="screen shot 2017-05-19 at 23 24 10" src="https://cloud.githubusercontent.com/assets/4672155/26267459/893850d8-3cea-11e7-8eef-6ac9661c6c69.png">
